src/components/general/Card.js

```diff
@@ -1,8 +0,0 @@
-import React from "react";
-import "./Card.css";
-
-function Card(props) {
```

Comment:
Instead of doing `props.children` and `props.class` you can instead do:

const Card = ({ class, children }) => {
...

and use the variables directly, can prevent you from typing props a milion times

Reply:
Also use arrow function, not function